doc/api/process.md

```diff
@@ -1684,6 +1684,21 @@ Node.js process.
 console.log(`Report directory is ${process.report.directory}`);
 ```
 
+### process.report.filename
```

Comment:
The sad thing is that I tried to put this in the right place but can't alphabet apparently. Sorry

Reply:
You've got too much on your plate to be sorry about missing nits))